src/main/scala/de/upb/cs/swt/delphi/instanceregistry/authorization/AuthProvider.scala

```diff
@@ -128,6 +172,13 @@ class AuthProvider(authDAO: AuthDAO) extends AppLogging {
     }
   }
 
+  def refrestTokenRequire(credentials: Credentials, userType: UserType = UserType.Admin) : Option[Number] = {
```

Comment:
You do not actually need this method. It does nothing with the ```userType``` argument and only calls the ```checkRefreshToken``` method. You can remove this method and in the ```Server.scala``` line 1308 replace ```handler.authProvider.refrestTokenRequire(_, userType = UserType.Admin)``` with ```handler.authProvider.checkRefreshToken(_)```.